units/systemd-firstboot.service

```diff
@@ -31,7 +31,7 @@ Before=shutdown.target
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=systemd-firstboot --prompt-locale --prompt-timezone --prompt-root-password
+ExecStart=systemd-firstboot --prompt
```

Comment:
This doesn't belong to the commit. Please split it out and explain the rationale separately.